### DISCUSSION TITLE
Organizing Expenses and Income Categories

### DISCUSSION THREAD

**Discussion by Vertelleus** [emavgl/oinkoin#163] [Ideas]
### Current state
Expenses and Income categories are listed in grid format based on the first created to the last created.

### Suggestion
Allow the user to long press on a category and drag it to another position, this allowing the user to reorganize the order of how the categories are shown. 

### Advanced Suggestion
Maybe grouping categories somehow, i.e. grouping the Medical visit category with the prescription and health care payments. (I guess users could do this by color)
Or allowing the category order shown based on last used or most used.

I hope this is well received. Thanks for the consideration.

**Comment by emavgl** [comment]
Hi, the sorting of the category is almost ready for release. Would you like to try this out and give me some feedback? If yes, you can download and install the apk from here. https://github.com/emavgl/oinkoin/pull/166

**Reply by Vertelleus** [reply]
Tested this and on Samsung Fold 4. Work great!
Sorry, thought I sent a reply 4 days ago, guess it didn't send.
